src/components/20-molecules/accordeon/story.js

```diff
@@ -0,0 +1,56 @@
+import { boolean, text, withKnobs } from '@storybook/addon-knobs';
+import { html, render } from 'lit-html';
+import './index';
+import readme from './README.md';
+import changelog from './CHANGELOG.md';
+
+export default {
+  title: 'Components/Accordeon',
+  decorators: [withKnobs],
+  parameters: {
+    readme,
+    usage: {
+      componentName: 'accordeon',
+    },
+    changelog,
+  },
+};
+
+export const Accordeon = () => {
+  const disabled = boolean('disabled', false);
+  const open = boolean('open', false);
+  const title = text('title', 'This is the title of the accordeon');
+  const small = boolean('small', false);
+  const icon = text(
+    'icon',
+    '<svg viewBox="0 0 96 96" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M62 85c-3-2-6-6-6-14a27.46 27.46 0 016.13-17.54S61.66 69.52 66 71c4 1 2-10 2-10s7 3.28 7 11c-.16 4.72-1 9-5 13 11 0 17-15 17-28 0 0-4 8-6 6s2-6 2-12a21 21 0 00-5.67-13.75C79 46.45 69 53 69 53s5-19-7.46-29c2.46 16-6.2 16.06-7.78 25.47 0 0-4.25-3.39-4.84-10.62C39 50 52 72 48 70c-2-1-4-3-6-7-.94 17 16 22 20 22z" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path d="M12.64 40.39V75h32.24" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path d="M30.82 75V48.58h13.79M60.2 36.56L39.55 20.94a.9.9 0 00-1.1 0L13.12 40.1A2.57 2.57 0 019 38.05a2.59 2.59 0 011-2l26.8-20.31a3.63 3.63 0 014.4 0l20.37 15.41-1.37 5.41z" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path d="M39 39.59c2.01 0 3.64-1.625 3.64-3.63s-1.63-3.63-3.64-3.63a3.635 3.635 0 00-3.64 3.63c0 2.005 1.63 3.63 3.64 3.63z" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>'
+  );
+
+  const wrapper = document.createElement('div');
+
+  const template = html`
+    <axa-accordeon
+      ?small="${small}"
+      ?open="${open}"
+      ?disabled="${disabled}"
+      .icon="${icon}"
```

Comment:
Was it necessary to add a `.` here? I know that this has to be something from Markus, but I know that we had problems with those "property-only" attributes in the past.

Reply:
It does work without the `.`. Should I remove it?